doc/external-libs/petsc.html

```diff
@@ -113,7 +113,12 @@
 
     <p>
       This automatically builds PETSc with both MPI and the algebraic
-      multigrid preconditioner package Hypre (which we use in step-40).
+      multigrid preconditioner package Hypre (which we use in step-40). If you
+      would like to use PETSc with MPI then we recommend that you install it
+      through your package manager instead of letting PETSc install it: put
+      another way, installing PETSc with the flag <code>--download-mpich</code>
+      sometimes causes problems (such as linking errors or poor performance)
```

Comment:
Maybe say "often" instead of "sometimes"? I'd really like to discourage people from using it, if necessary by overstating the problems.